contribute.html

```diff
@@ -16,4 +16,29 @@
   {% include contribute-card.html icon="fa-pencil-ruler" url="https://github.com/AntennaPod/AntennaPod/labels/needs:%20mock-up%20or%20user%20story" item="design" target="external" %}
 </div>
 
-<p id="donate">{% t contribute.donate %}</p>
+<p id="donate">{% t contribute.donate.link %}</p>
+
+<div class="modal fade" id="donationModal" tabindex="-1" role="dialog" aria-labelledby="donationModalLabel" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="donationModalLabel">{% t contribute.donate.modal-title %}</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        {% t contribute.donate.modal-text %}
+        <ul>
+          <li>{% t contribute.donate.modal-text-item1 %}</li>
+          <li>{% t contribute.donate.modal-text-item2 %}</li>
+        </ul>
```

Comment:
Can't we somehow move this to a markdown file, so that it can be translated as a whole instead of creating many individual strings?